src/App.stories.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+/** @jsxRuntime classic */
+/** @jsx jsx */
```

Comment:
The [Emotion docs imply](https://5faaafd0bd0f3f0008469537--emotion.netlify.app/docs/css-prop#get-started) that if you're using the css prop preset, you shouldn't need to use the JSX pragma as well. Have you found this _not_ to be the case?